src/storage/heap_file.c

```diff
@@ -6742,6 +6742,7 @@ heap_scancache_start_internal (THREAD_ENTRY * thread_p, HEAP_SCANCACHE * scan_ca
 
   scan_cache->page_latch = S_LOCK;
 
+  scan_cache->node.classname = NULL;
```

Comment:
I'd like to set classname to open scan cache.
`heap_scancache_start_internal` calls `heap_get_hfid_and_file_type_from_class_oid` and `heap_get_hfid_from_class_record` retrieves class record. You may fetch `classname` as well as `hfid`.
Advantage of the idea is other functions (`locator_update_force`) also save calls to get class name.